src/app/fyle/personal-cards/personal-cards.page.ts

```
@@ -415,13 +406,13 @@ export class PersonalCardsPage implements OnInit, AfterViewInit {
   }
 
-  onCardChanged(card: PersonalCard): void {
-    this.selectedAccount = card;
+  onCardChanged(event: string): void {
+    this.selectedAccount = event;
     this.acc = [];
     const params = this.loadData$.getValue();
     const queryParams = params.queryParams || {};
     queryParams.btxn_status = `in.(${this.selectedTransactionType})`;
-    queryParams.ba_id = 'eq.' + this.selectedAccount.id;
+    queryParams.ba_id = 'eq.' + this.selectedAccount;
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)